salt/etc-default/dqt_celery

```diff
@@ -0,0 +1,30 @@
+CELERYD_NODES="{{ nodes }}"
```

Comment:
Should this file be called pwfy_dqt_celery? Just to separate from others "dqt's" that may come along?